pointpats/spacetime.py

```diff
@@ -612,6 +611,69 @@ def modified_knox(s_coords, t_coords, delta, tau, permutations=99):
     modknox_result = {'stat': stat, 'pvalue': pvalue}
     return modknox_result
 
-if __name__ == "__main__":
-    import doctest
-    doctest.testmod()
\ No newline at end of file
+def _shuffle_matrix(X, ids):
```

Comment:
It looks here like `_shuffle_matrix()` is an exact duplicate of the one below `_get_lower()`. Is this intended or a mistake?

Reply:
It was a mistake. Thanks for spotting it!